src/partials/benefits/benefits.html

```diff
@@ -1 +1,67 @@
-<section class="benefits">BENEFITS</section>
+<section class="benefits">
+  <div class="container">
+    <div class="benefits-wrapper">
+      <h2 class="benefits-section-title">BENEFITS OF WORKING WITH ME</h2>
+      <ul class="benefits-list">
+        <li class="benefits-items-list">
+          <div class="icon-wrapper">
+            <svg class="benefits-icon" width="24" height="24">
+              <title>Expertise</title>
+              <use href="/assets/svg/icons.svg#icon-user"></use>
+            </svg>
+          </div>
+          <h5 class="header-5 title">Expertise</h5>
+          <p class="main-text-with-opacity">
+            As a highly experienced developer, I have deep knowledge of
+            programming and website development.
+          </p>
+        </li>
+        <li class="benefits-items-list">
+          <div class="icon-wrapper">
+            <svg class="benefits-icons" width="24" height="24">
```

Comment:
to improve accessability you can add to SVG title and description 
Example below
<svg
	 xmlns="http://www.w3.org/2000/svg"
	 xml:lang="en" lang="en">
   <title>The New EPUB Logo</title>
   <desc>
	  The EPUB logo is a lower-case letter E that has 
	  been tilted 45 degrees to counter-clockwise so 
	  that it appears to be sitting in balance on its 
	  lower-left corner. The E is drawn as a single, 
	  unclosed line of green starting at the centre 
	  of the image, moving to the outer edge and then 
	  continuing around in a box-like pattern. The
	  external corners have all been rounded.
   </desc>
<use></use>
</svg>